MusicTaste/MusicTaste/View Controllers/ProfileViewController.m

```diff
@@ -1,13 +1,30 @@
 #import "ProfileViewController.h"
+#import "UIImageView+AFNetworking.h"
+#import "User.h"
 
 @interface ProfileViewController () 
+//connects objects: Screenname and profileview
```

Comment:
This comment is unnecessary since IBOutlet is always used to connect objects to xibs/storyboards. (IB stands for Interface Builder.)